.github/workflows/build.yml

```diff
@@ -167,23 +177,24 @@ jobs:
 
       - uses: Swatinem/rust-cache@v1
 
-      - name: Install Dependencies
-        run: |
-          sudo apt-get install -y curl gnupg ca-certificates git gcc-multilib g++-multilib cmake libssl-dev pkg-config libfreetype6-dev libasound2-dev libexpat1-dev libxcb-composite0-dev libbz2-dev freeglut3-dev libxi-dev
-
       - name: Check Formatting
         run: |
           cargo fmt --all -- --check
 
-      - name: Install Neovim
+      - name: Install dependencies and neovim

```

Comment:
Here i optimized three steps, `update/install deps/install neovide` into one. It should be faster this way.